docs/security.md

```diff
@@ -3,96 +3,12 @@ post_title: Security
 menu_order: 40
 enterprise: 'no'
 ---
-# Mesos Security
 
-## SSL
-
-<table class="table">
```

Comment:
@mgummelt  spark-auth does "live" with this content - once it's finally rendered at docs.mesosphere.com/service-docs/spark/. do you think we should embed it directly in the service guide repo?